libsolidity/formal/SymbolicState.cpp

```diff
@@ -126,7 +126,7 @@ smtutil::Expression SymbolicState::txMember(string const& _member) const
 	return m_tx.member(_member);
 }
 
-smtutil::Expression SymbolicState::txConstraints(FunctionDefinition const& _function) const
+smtutil::Expression SymbolicState::txConstraints(FunctionDefinition const* _function) const
```

Comment:
Not sure I understand the motivation for this change.

Reply:
The idea is that I need to obtain `txConstraints` also for the deployment phase, and also for the situation where no explicit constructor is present, because the transaction data might be used in implicit constructor using direct initiatilization of the state variables.
In that case,  there is no `FunctionDefinition` to pass to this method, so I wanted to allow passing `nullptr` for this case.

Reply:
Okay, was thinking if an assert on `_function` is needed but I just saw `_function && ..` in the if clause so that should take care of it. I didn't find any potential for nullptr deref.